api/api_full.go

```diff
@@ -689,7 +689,15 @@ type FullNode interface {
 	// MethodGroup: Paych
 	// The Paych methods are for interacting with and managing payment channels
 
-	PaychGet(ctx context.Context, from, to address.Address, amt types.BigInt) (*ChannelInfo, error)                     //perm:sign
+	// PaychGet gets or creates a payment channel between address pair
```

Comment:
If I am reading the code correctly, we have two bools, but actually one of the combinations is invalid and always errors (reserve = false, off chain = true). I wonder if instead of two bools, this should just be some kind of typed integer to represent the 3 available modes that one can actually use. (on-chain no reserve, on-chain reserve, off-chain reserve)